xbee_tx_block.py

```diff
@@ -20,11 +21,17 @@ class XBeeTX(XBeeBase):
 
     version = VersionProperty(version='0.2.1')
     data = Property(title="Data", default="{{ $.to_dict() }}")
-    # todo: make dest_addr a block property
+    dest_addr = Property(title='Destination Address \
+                         (2 or 8 bytes hex, ex: "00 05")',
+                         default='')
+    # todo: should None be allowed? if not then line33 can omit the if/else
```

Comment:
Lets get this answered before merging this. I'm not totally sure what the debate is here and don't want this semi-vague `todo` in the code.